orchestrator/services/tasks.py

```diff
@@ -136,7 +136,7 @@ def __init__(self) -> None:
         self.number_of_workers_online = len(stats)
 
         def sum_items(d: dict) -> int:
-            return sum(len(l) for _, l in d.items())
+            return sum(len(l) for _, l in d.items()) if d else 0
```

Comment:
not sure why the 'if d' is needed now. Empty dict should be fine.

Also you could probably use:

return sum(len(l) for l in d.values())